core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -899,6 +899,12 @@
     sa.deck.app.ci.config:
       bw_item: build_farm
       attachment: sa.deck.app.ci.config
+    sa.config-updater.app.ci.config:
+      bw_item: build_farm
+      attachment: sa.config-updater.app.ci.config
+    sa.config-updater.api.ci.config:
```

Comment:
Not generated yet.

/hold

Reply:
Uploaded to BW.
Since we are moving away from api.ci.
I did not store the script to generate it (i modified the one for build01).